txscript/script_test.go

```diff
@@ -3844,6 +3848,106 @@ func TestGetPreciseSigOps(t *testing.T) {
 	}
 }
 
+// mustDecodeHex attempts to decode the passed hex-encoded string. In the case
+// that a string is unable to be decoded, then this function will panic. This
+// helper function is meant to be used in the creation of table driven tests.
+func mustDecodeHex(s string) []byte {
```

Comment:
That said, I actually do like the `mustDecodeHex` name better.  Its name is much more in line with many other funcs that panic.  We probably should (in a separate PR) change that.